edsdme/blocks/partners-navigation/utilities/utilities.js

```diff
@@ -393,3 +393,8 @@ export const [setUserProfile, getUserProfile] = (() => {
     () => profilePromise,
   ];
 })();
+
+export const getMainNavItems = (content, features) => {
+  return [...content.querySelectorAll('h2, p:only-child > strong > a, p:only-child > em > a')]
```

Comment:
@richardhand @zagi25 is this DOM structure defined by the code we copied from Milo or can this potentially break when Milo changes something on their side?

Reply:
@Ben-Zahler the same DOM structure is queried for in the copied gnav code. So this would break already if they change the item API for the menu items in the global nav, which seems unlikely but could happen I guess.

Reply:
@richardhand I guess the worst thing that can happen here is that personalization does not get applied, right? As long as the navigation basically is still working, this should be fine